examples/container-service-extension/v4.1/install/step2/3.11-cse-install-7-cse-server-init.tf

```diff
@@ -0,0 +1,105 @@
+# ------------------------------------------------------------------------------------------------------------
+# CSE v4.1 installation:
+#
+# * Please read the guide present at https://registry.terraform.io/providers/vmware/vcd/latest/docs/guides/container_service_extension_install
+#   before applying this configuration.
+#
+# * Rename "terraform.tfvars.example" to "terraform.tfvars" and adapt the values to your needs.
+#   Other than that, this snippet should be applied as it is.
+#   You can check the comments on each resource/data source for more help and context.
+# ------------------------------------------------------------------------------------------------------------
+
+# Login into VCD with the cse_admin username created above. This will be used to provision
+# an API token that must be consumed by the CSE Server.
+# This should not be changed.
+provider "vcd" {
+  alias                = "cse_admin"
+  url                  = "${var.vcd_url}/api"
+  user                 = var.cse_admin_username
+  password             = var.cse_admin_password
+  auth_type            = "integrated"
+  org                  = var.administrator_org
+  allow_unverified_ssl = var.insecure_login
+  logging              = true
+  logging_file         = "cse_install_cse_admin.log"
+}
+
+# Generates an API token for the CSE Admin user, that will be used to instantiate the CSE Server.
+# This should not be changed.
+resource "vcd_api_token" "cse_admin_token" {
+  provider         = vcd.cse_admin
+  name             = "CSE Admin API Token"
+  file_name        = var.cse_admin_api_token_file
+  allow_token_file = true
+}
+
+data "local_file" "cse_admin_token_file" {
+  filename = vcd_api_token.cse_admin_token.file_name
+}
+
+# This is the CSE Server vApp
+resource "vcd_vapp" "cse_server_vapp" {
+  org  = vcd_org.solutions_organization.name
+  vdc  = vcd_org_vdc.solutions_vdc.name
+  name = "CSE Server vApp"
+
+  lease {
+    runtime_lease_in_sec = 0
+    storage_lease_in_sec = 0
+  }
+}
+
+# The CSE Server vApp network that will consume an existing routed network from
+# the solutions organization.
+resource "vcd_vapp_org_network" "cse_server_network" {
+  org = vcd_org.solutions_organization.name
+  vdc = vcd_org_vdc.solutions_vdc.name
+
+  vapp_name        = vcd_vapp.cse_server_vapp.name
+  org_network_name = vcd_network_routed_v2.solutions_routed_network.name
+
+  reboot_vapp_on_removal = true
+}
+
+# The CSE Server VM. It requires guest properties to be introduced for it to work
+# properly. You can troubleshoot it by checking the cse.log file.
```

Comment:
Should we mention here how to find the cse.log?

Reply:
I will change this sentence to just refer to their docs (instead of mentioning any log) as it feels that this information belongs to them (for example, they could change the log name and we should not be impacted).